src/main/java/Duke.java

```diff
@@ -1,10 +1,261 @@
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    public static void main(String[] args) throws DukeException, FileNotFoundException {
+        Database database = new Database( "data.txt");
+        ArrayList<String> listOfTasks;
+        ArrayList<Task> myList;
+       try {
+            listOfTasks = database.readFile();
+            myList = readInput(listOfTasks);
+        }
+        catch (FileNotFoundException e){
+            throw new FileNotFoundException("No File Detected");
+        }
+        System.out.println("____________________________________");
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println("____________________________________");
+
+        Scanner input = new Scanner(System.in);
+        while(input.hasNextLine()) {
+            String s = input.next();
+            switch (s){
+                case "todo":
+                    System.out.println("____________________________________");
+                    try {
+                        String s1 = input.nextLine();
```

Comment:
You might have missed this: variable names could do with a more specific name, especially since the scope isn't small. Perhaps `s1` can be renamed as `userInputLine`.